public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
```

Comment:
Add a very short comment at the top describing the rules of the game or how to play.  A developer who knows the game will find it easier to navigate your code, many people will never have heard of 'Simon Says'.